Libraries/PeriphDrivers/Source/WDT/wdt_me17.c

```diff
@@ -153,3 +153,47 @@ void MXC_WDT_ClearIntFlag(mxc_wdt_regs_t *wdt)
 {
     MXC_WDT_RevB_ClearIntFlag((mxc_wdt_revb_regs_t *)wdt);
 }
+
+int MXC_WDT_SetClockSource(mxc_wdt_regs_t *wdt, mxc_wdt_clock_t clock_source)
+{
+    const uint8_t clock_source_num = 8;
+    uint8_t idx = 0;
+    uint8_t instance = 0;
+
+#if TARGET_NUM == 32655 || TARGET_NUM == 78000
+    mxc_wdt_clock_t clock_sources[2][8] = {
+        { MXC_WDT_PCLK, MXC_WDT_IBRO_CLK, 0xFF, 0xFF, 0xFF, 0xFF, 0xFF, 0xFF },
+        { MXC_WDT_IBRO_CLK, MXC_WDT_ERTCO_CLK, MXC_WDT_INRO_CLK, 0xFF, 0xFF, 0xFF, 0xFF, 0xFF }
+    };
+#elif TARGET_NUM == 32680
+    mxc_wdt_clock_t clock_sources[2][8] = {
+        { MXC_WDT_PCLK, MXC_WDT_IBRO_CLK, 0xFF, 0xFF, 0xFF, 0xFF, 0xFF, 0xFF },
+        { MXC_WDT_IBRO_CLK, 0xFF, MXC_WDT_INRO_CLK, 0xFF, 0xFF, 0xFF, 0xFF, 0xFF }
+    };
+#else
+    mxc_wdt_clock_t clock_sources[2][8] = { { 0xFF, 0xFF, 0xFF, 0xFF, 0xFF, 0xFF, 0xFF, 0xFF },
```

Comment:
Throw a build error here instead with `#error ...`.  This will help us catch cases where this file is reused in the future.

Reply:
Updated code for ME15 and ME17.